build/ci/github-actions/index.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Docker GitHub Actions
+description: >
+  Docker maintains a set of official GitHub Actions for building Docker images.
+keywords: github, actions, gha, ci, build
+redirect_from:
+- /ci-cd/github-actions/
+---
+
+GitHub Actions is a popular CI/CD platform for automating your build, test, and
+deployment pipeline. Docker provides a set of official GitHub Actions for you to
+use in your workflows. These official actions are reusable, easy-to-use
+components for building, annotating, and pushing images.
+
+The following GitHub Actions are available:
+
+- [Build and push Docker images](https://github.com/marketplace/actions/build-and-push-docker-images){:
+  target="blank" rel="noopener" class=""}: build and push Docker images with
+  BuildKit.
+- [Docker Login](https://github.com/marketplace/actions/docker-login){:
+  target="blank" rel="noopener"}: sign in to a Docker registry.
+- [Docker Setup Buildx](https://github.com/marketplace/actions/docker-setup-buildx){:
+  target="blank" rel="noopener"}: Initiates a BuildKit builder
+- [Docker Metadata action](https://github.com/marketplace/actions/docker-metadata-action){:
+  target="blank" rel="noopener"}: Extracts metadata from Git reference and
+  GitHub events.
+- [Docker Setup QEMU](https://github.com/marketplace/actions/docker-setup-qemu){:
+  target="blank" rel="noopener"}: installs [QEMU](https://github.com/qemu/qemu)
+  static binaries for multi-arch builds.
+- [Docker Buildx Bake](https://github.com/marketplace/actions/docker-buildx-bake){:
+  target="blank" rel="noopener"}: enables using `docker buildx bake`.
+
+Using Docker's actions provides an easy-to-use interface, while still allowing
+flexibility for customizing build parameters.
+
+## What's next
+
+- Learn how to get started with actions by reading the [GitHub Actions tutorial](./gha-tutorial.md)
```

Comment:
`tutorial.md` instead of `gha-tutorial.md` or maybe `getting-started.md`?

Reply:
Hum I wonder if we should not merge `Docker GitHub Actions` and `Tutorial` page an name it `Overview` actually?